railties/lib/rails/generators/app_base.rb

```diff
@@ -558,6 +558,9 @@ def dockerfile_deploy_packages
         # ActiveRecord databases
         packages << deploy_package_for_database unless skip_active_record?
 
+        # Add curl to work with the default healthcheck strategy in MRSK
+        packages << "curl"
```

Comment:
@dhh looks like the "curl" is doubled now as it was present already on https://github.com/rails/rails/pull/48282/files#diff-a83245368df35c610a37260d0a51041de89211857d47cfd0b2d1301a11709164R556

Reply:
Lol. I should have checked that first!